src/Router.js

```diff
@@ -85,62 +70,119 @@ export default class Router {
       throw new Error('Router#registerRoutes must be passed an array of Routes')
     }
     routes.map(route => {
```

Comment:
it's kind of silly this was a `map` before, probably more semantic to be a `forEach`